<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpHistoryCopyDialog</class>
 <widget class="QDialog" name="OpHistoryCopyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy stack history</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>591</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>782</width>
      <height>582</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="opaqueResize">
     <bool>true</bool>
    </property>
    <property name="childrenCollapsible">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <layout class="QVBoxLayout" name="filterLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <layout class="QFormLayout" name="optionsLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="stackTargetLabel">
          <property name="text">
           <string>Apply to:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="StackSelectorWidgetView" name="stackTargetSelector"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="operationsContainer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Operations:</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="filterPropertiesLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="previewPropertiesContainer">
          <property name="title">
           <string>Preview</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="previewAutoUpdate">
             <property name="text">
              <string>Auto Update</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updatePreviewButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Update Now</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="previewImageIndexLabel">
             <property name="text">
              <string>Image:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="previewImageIndex"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="stackSourceLabel">
          <property name="text">
           <string>Operations from:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="StackSelectorWidgetView" name="stackSourceSelector"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <spacer name="buttonSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Apply To Stack</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <layout class="QVBoxLayout" name="previewsPaneLayout">
      <item>
       <layout class="QHBoxLayout" name="previewsPaneToolbar">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item alignment="Qt::AlignRight|Qt::AlignTop">
         <widget class="QCheckBox" name="showHistogramLegend">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Show Legend</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight|Qt::AlignTop">
         <widget class="QCheckBox" name="combinedHistograms">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>Combine histograms</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="previewsLayout"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StackSelectorWidgetView</class>
   <extends>QComboBox</extends>
   <header>mantidimaging.gui.widgets.stack_selector</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
